python-sdk/src/astro/files/base.py

```diff
@@ -145,7 +145,7 @@ def resolve_file_path_pattern(
     location = create_file_location(path_pattern, conn_id)
 
     files = [
-        File(
+        File(  # type: ignore
```

Comment:
hmm why?

Reply:
Removed this https://github.com/astronomer/astro-sdk/pull/797/commits/cedb4a4c681d9fd7643eb5352b4543d16cd3a255